site/content/articles/monitoring-multiple-openshift-clusters.adoc

```diff
@@ -0,0 +1,74 @@
+---
+title: "Monitoring Multiple Openshift Clusters"
+date: 2019-11-08T08:05:32+01:00
+authors: ["redhat-cop"]
+categories:
+  - openshift
+  - operations
+tags:
+  - research
+draft: true
+---
+= Monitoring Multiple Openshift Clusters
+Andrea Ferraresi <andrea.ferraresi@redhat.com>
+:toc: macro
+:toc-title:
+
+include::site/layouts/variables.adoc[]
+
+How to deal with monitoring of multiple Openshift Cluster: the challenges and the different approaches on scaling 
+
+toc::[]
+
+== Introduction to prometheus
+We all know Prometheus to be a very simple, very easy to use, piece of software:
```

Comment:
I'd avoid using 'very' twice in quick succession particularly when both statements are implying the same thing. How about 'We all know Prometheus to be a very accessible piece of software'